scripts/make-changelog.js

```diff
@@ -0,0 +1,63 @@
+const shell = require('shelljs');
+const request = require('request');
+require('dotenv').config();
+
+const GITHUB_API_URL = 'https://api.github.com'
+
+const usage = () => {
+    console.error(`Usage: node ${process.argv[1]} <last-release-tag>
+
+Environment:
+    GITHUB_TOKEN    Github API token`);
+    process.exit(1);
+}
+
+if (process.argv.length !== 3 || !process.env.GITHUB_TOKEN) {
+    usage();
+}
+
+const lastRelease = process.argv[2];
+const githubToken = process.env.GITHUB_TOKEN;
+
+const listPRsSince = (commit) => {
+    return shell
+        .exec(`git log --merges --first-parent master --pretty=format:"%s" ${commit}..`, { silent: true })
+        .split(/\n/)
+        .map((msg) => msg.match(/Merge pull request #(\d+)/)[1]);
```

Comment:
I got an error when running the script with `1.1.0`, where no PR is merged since the tag, because `.split(/\n/)` returns [""]. You may want to check the length of `msg.match(/Merge pull request #(\d+)/)'`.